src/Widgets/ImageWidget/DimensionsEditor.tsx

```diff
@@ -97,3 +98,49 @@ const DimensionEditor = connect(function DimensionEditor({
     if (value !== '') widget.update({ [attribute]: `${value}${newUnit}` })
   }
 })
+
+const ObjectFit = connect(function ObjectFit({
+  widget,
+}: {
+  widget: ImageWidgetInstance
+}) {
+  const readOnly = !canEdit(widget.obj()) || isComparisonActive()
```

Comment:
`readOnly` now gets calculated twice. I suggest to move it one component up and pass it on as a prop.